exonum-java-binding-core/rust/src/runtime/config.rs

```diff
@@ -34,3 +38,31 @@ pub struct ServiceConfig {
     /// A port of the HTTP server for Java services. Must be distinct from the ports used by Exonum.
     pub port: i32,
 }
+
+/// Error returned while validating user-specified additional parameters for JVM.
+#[derive(Debug, Clone, Copy)]
+pub enum UserParameterError {
+    /// Trying to specify a parameter that is set by EJB internally.
+    ForbiddenParameter,
```

Comment:
How is that displayed to the user? Can we include a parameter that they shall use instead in the error message?

Reply:
Good idea, I will expand this idea